plugin/trino-http-event-listener/src/main/java/io/trino/plugin/httpquery/HttpEventListener.java

```diff
@@ -125,19 +125,33 @@ public void onSuccess(StatusResponse result)
                                 verify(result != null);
 
                                 if (result.getStatusCode() >= 500 && attempt < config.getRetryCount()) {
-                                    attemptToSend(request, attempt + 1, nextDelay(delay));
+                                    Duration nextDelay = nextDelay(delay);
+                                    int nextAttepmt = attempt + 1;
+
+                                    log.warn("QueryId = \"%s\", attempt = %d/%d, URL = %s | Ingest server responded with code %d, will retry after approximately %d seconds",
+                                            queryId, attempt + 1, config.getRetryCount() + 1, request.getUri().toString(),
+                                            result.getStatusCode(), nextDelay.roundTo(TimeUnit.SECONDS));
+
+                                    attemptToSend(request, nextAttepmt, nextDelay, queryId);
                                     return;
                                 }
 
                                 if (!(result.getStatusCode() >= 200 && result.getStatusCode() < 300)) {
-                                    log.error("Received status code %d from ingest server URI %s; expecting status 200", result.getStatusCode(), request.getUri());
+                                    log.warn("QueryId = \"%s\", attempt = %d/%d, URL = %s | Received status code %d from ingest server; expecting status 200",
+                                            queryId, attempt + 1, config.getRetryCount(), request.getUri().toString(),
+                                            result.getStatusCode(), request.getUri().toString());
+                                    return;
                                 }
+
+                                log.info("QueryId = \"%s\", attempt = %d/%d, URL = %s | Query event delivered successfully",
```

Comment:
feels to verbose. I think it should be `debug`

Reply:
I see you changed that in latter commit but it should be `debug` already in "Add more logging to the HTTP event listener"

Reply:
Please do. We want clean commit history when possible. I will merge when CI passes.